src/v2/Apps/Artist/Components/ArtistMetaCanonicalLink.tsx

```diff
@@ -6,27 +6,43 @@ import { ArtistMetaCanonicalLink_artist } from "v2/__generated__/ArtistMetaCanon
 import { hasSections as showMarketInsights } from "v2/Apps/Artist/Components/MarketInsights/MarketInsights"
 import { hasOverviewContent } from "v2/Apps/Artist/Components/NavigationTabs"
 
-export const canonicalPath = (artist: ArtistMetaCanonicalLink_artist) => {
+export const computeCanonicalPath = (
+  artist: ArtistMetaCanonicalLink_artist,
+  path: string
+) => {
+  const basePath = `/artist/${artist.slug}`
+  const pathParts = [basePath]
```

Comment:
I wonder if it would make sense to make the full path available in these global helpers rather then constructing it from the component: https://github.com/artsy/force/blob/master/src/lib/getPageType.ts